modernism/templates/base.html

```diff
@@ -44,6 +44,8 @@
 
   {% include "includes/footer.html" %}
 
+  <button onclick="topFunction()" id="myBtn" title="Go to top">&#10148;</button>
```

Comment:
Of course this works. But you could try as a challenge and for clean code reasons to add a click event and define what should happen in the js file and not call the function directly in the html.